app/controllers/collaborations_controller.rb

```diff
@@ -0,0 +1,38 @@
+class CollaborationsController < ApplicationController
+  helper_method :mailbox, :conversation
```

Comment:
I don't think you need to repeat the helper_method declaration here - it's already in the base class